lib/config.js

```diff
@@ -631,6 +631,7 @@ const tryToLoad = (configPath, fallbackHandler) => {
   try {
     pkg = require(sysPath.resolve('.', 'package.json'));
   } catch (e) {
+    logger.error(e);
```

Comment:
We don't actually need this logged. We are loading package.json here simply to try and recover from brunch-config require errors